runtime/util/hshelp.c

```diff
@@ -3229,6 +3230,18 @@ classIsModifiable(J9JavaVM * vm, J9Class * clazz)
 		rc = JNI_FALSE;
 	} else if ((J2SE_VERSION(vm) >= J2SE_19) && J9_ARE_ALL_BITS_SET(clazz->classFlags, J9ClassIsAnonymous)) {
 		rc = JNI_FALSE;
+	} else {
+		/* Handle name based matches */
+		U_8* className = J9UTF8_DATA(J9ROMCLASS_CLASSNAME(clazz->romClass));
+		U_8 classNameLength = J9UTF8_LENGTH(className);
+
+		switch(classNameLength) {
+			case 22:
```

Comment:
Nitpick: Our typical coding convention is to line the case up with the switch and to always have a break statement

```
switch(classNameLength) {
case 22:
	if (strncmp((char*)className, "com/ibm/jit/JITHelpers", 22) == 0) {
	......
	break;
default:
       rc = JNI_TRUE;
}
```